Backend/admin/src/screens/messages/message-form.js

```diff
@@ -1,33 +1,30 @@
-import './message-form.css';
 import React from 'react';
 import { useHistory } from 'react-router';
 import PropTypes from 'prop-types';
-import { Button, Form, Input } from 'antd';
+import { Button, Card, Form, Input, Row } from 'antd';
 
 const MessageForm = ({ form, initialValues, onFinish, submitTitle }) => {
   const history = useHistory();
   const values = initialValues
     ? {
-      body: initialValues.body,
-      from: initialValues.from,
-      to: initialValues.to,
-    }
+        body: initialValues.body,
+        from: initialValues.from,
+        to: initialValues.to,
+      }
     : {};
 
   const onCancel = () => {
     history.goBack();
   };
 
-  return(
-    <div className="message-form-wrapper">
-      <div className="message-form">
+  return (
+    <div className="form-wrapper">
+      <Card title="Adauga un mesaj">
         <Form
           initialValues={values}
```

Comment:
Same here on edit. If I am waiting 2 mins for the message to be retrieved ? You are rendering null which means I don't know if the request is in pending. 
Can you also add here the skeleton with the animation , this is letting the user know that some request is pending.